docs/how/search.md

```diff
@@ -0,0 +1,7 @@
+# How to Search
+
+This guide is supposed to answer questions related to search capabilities of DataHub. You can enter any of the examples into search bar of DataHub and see the results.
+
```

Comment:
Nit we should put these under different subheads... Also we can show examples for tags, terms, and platforms I think ..